corehq/apps/cloudcare/static/cloudcare/css/formplayer.css

```diff
@@ -4,4 +4,23 @@
 
 .module-icon {
     max-height: 100px;
-}
\ No newline at end of file
+}
+
+body {
+    margin: 4px;
+}
+
+.grid-wrapper {
+    display: grid;
+    grid-template-columns: repeat(12, 60px) min-content 60px;
```

Comment:
regarding the 60px, how much do you care about / have you tested different screen sizes? should this set of styles vary based on screen size? (see https://github.com/dimagi/commcarehq-prelogin/blob/f4d75e55275b530145ef9ddc07c9b51ddcb660d1/static/prelogin/less/components/home.less#L9 for an example of targeting different screen sizes)